network-functions/core-network/charts/astri-upf/templates/deployment.yaml

```diff
@@ -0,0 +1,142 @@
+# SPDX-License-Identifier: Apache-2.0
+# Copyright (c) 2021 Astri Corporation
+
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ include "upf.fullname" . }}
+  labels:
+    env: test
+spec:
+  replicas: {{ .Values.upf.replicaCount }}
+  selector:
+    matchLabels:
+      env: test
+      app: upf-agent
+  template:
+    metadata:
+      annotations:
+        k8s.v1.cni.cncf.io/networks: {{"'["}}
+        {{- " " -}}
+        {{- $length := len .Values.upf.networks }}
+        {{- $found := false }}
+        {{- range $index, $network := .Values.upf.networks -}}
+          {{- if not $network.device -}}
+          {{- if eq $found true -}} {{- "," -}} {{- end -}}
+          {"name": {{- $network.name | quote -}}}
+          {{- end -}}
+        {{- end -}}
+        {{"]'"}}
+      labels:
+        env: test
+        app: upf-agent
+    spec:
+      initContainers:
```

Comment:
Does the UPF pod execute as a root user or non-root user (sudo user). The expectation is that the UPF POD should execute as non-root user.

Reply:
add security context to run UPF as non-root user at 98619e3.